templates/template-dominator/src/app.rs.lte

```diff
@@ -0,0 +1,124 @@
+use std::sync::Arc;
+
+use serde::{Deserialize, Serialize};
+use serde_wasm_bindgen::to_value;
+use wasm_bindgen::prelude::*;
+use wasm_bindgen_futures::spawn_local;
+use dominator::DomBuilder;
+pub use dominator::{clone, events, html, svg, with_node, Dom};
+pub use futures_signals::{
+    map_ref,
+    signal::{Mutable, Signal, SignalExt},
+    signal_vec::{MutableVec, SignalVec, SignalVecExt},
+};
+use web_sys::HtmlInputElement;
+
+#[wasm_bindgen]
+extern "C" {
+    #[wasm_bindgen(js_namespace = ["window", "__TAURI__", "{% if alpha %}core{% else %}tauri{% endif %}"])]
+    async fn invoke(cmd: &str, name: JsValue) -> JsValue;
+    
+    #[wasm_bindgen(js_namespace = ["window", "__TAURI__", "{% if alpha %}core{% else %}tauri{% endif %}"], js_name = invoke)]
+    async fn invoke2(cmd: &str);
+}
+
+#[derive(Clone, Serialize, Deserialize)]
+struct Args {
+    name: String,
+}
```

Comment:
Let's rename to GreetArgs so it can be consistent with other Rust templates

```suggestion
#[derive(Serialize, Deserialize)]
struct GreetArgs<'a> {
    name: &'a str,
}
```